include/NovelRT.Interop/Ecs/Audio/NrtAudioEmitterComponent.h

```diff
@@ -0,0 +1,19 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
+// for more information.
+#ifndef NOVELRT_INTEROP_ECS_AUDIO_AUDIOEMITTERCOMPONENT_H
+#define NOVELRT_INTEROP_ECS_AUDIO_AUDIOEMITTERCOMPONENT_H
+
+#include "../NrtEcsTypedefs.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+    void Nrt_AudioEmitterComponent_Update(void* lhs, const void* rhs, size_t size, void* context);
```

Comment:
Is there a reason the component pointers are `void*` if the component type is strongly known?

Reply:
Thought I needed void*s due to signature, but I was mistaken. Resolved.